Sources/PrimerSDK/Classes/PCI/Checkout Components/PrimerInputElements.swift

```diff
@@ -46,12 +46,12 @@ public class PrimerInputTextField: UITextField, PrimerHeadlessUniversalCheckoutI
         }
         set {
             super.text = newValue
-            self._text = newValue
+            self.internalText = newValue
         }
     }
 
     private var checkoutModulesTextFieldDelegate: PrimerHeadlessUniversalCheckout.Delegate?
-    internal var _text: String?
+    internal var internalText: String?
```

Comment:
`PrimerTextField` still uses the old pattern `_text`, I wonder if it should get the same treatment?

Reply:
I will have a double look at this part. Thanks!